library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt

```diff
@@ -485,7 +484,7 @@ class GroupTest {
 
         Thread.sleep(1000)
 
-        assertEquals(4, messageCallbacks)
+        assertEquals(secondMsgCheck, messageCallbacks)
```

Comment:
this test was failing before the change, think this last value is supposed to also be secondMsgCheck